[2025-05-14T11:07:16.519+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-13T23:44:36.782331+00:00 [queued]>
[2025-05-14T11:07:16.534+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-13T23:44:36.782331+00:00 [queued]>
[2025-05-14T11:07:16.537+0100] {taskinstance.py:2170} INFO - Starting attempt 4 of 5
[2025-05-14T11:07:16.562+0100] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_clean_reviews> on 2025-05-13 23:44:36.782331+00:00
[2025-05-14T11:07:16.571+0100] {standard_task_runner.py:60} INFO - Started process 73873 to run task
[2025-05-14T11:07:16.576+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'process_bank_reviews', 'run_clean_reviews', 'manual__2025-05-13T23:44:36.782331+00:00', '--job-id', '1900', '--raw', '--subdir', 'DAGS_FOLDER/post_processing.py', '--cfg-path', '/tmp/tmpdr3z4em5']
[2025-05-14T11:07:16.581+0100] {standard_task_runner.py:88} INFO - Job 1900: Subtask run_clean_reviews
[2025-05-14T11:07:16.685+0100] {task_command.py:423} INFO - Running <TaskInstance: process_bank_reviews.run_clean_reviews manual__2025-05-13T23:44:36.782331+00:00 [running]> on host DESKTOP-RRUTCI2.
[2025-05-14T11:07:16.885+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='process_bank_reviews' AIRFLOW_CTX_TASK_ID='run_clean_reviews' AIRFLOW_CTX_EXECUTION_DATE='2025-05-13T23:44:36.782331+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-13T23:44:36.782331+00:00'
[2025-05-14T11:07:16.890+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-14T11:07:16.892+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/airflow/dbt_project && dbt run --select clean_bank_reviews']
[2025-05-14T11:07:16.909+0100] {subprocess.py:86} INFO - Output:
[2025-05-14T11:07:51.453+0100] {subprocess.py:93} INFO - [0m10:07:51  Running with dbt=1.8.7
[2025-05-14T11:07:54.633+0100] {subprocess.py:93} INFO - [0m10:07:54  Registered adapter: postgres=1.8.2
[2025-05-14T11:08:05.211+0100] {subprocess.py:93} INFO - [0m10:08:05  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-05-14T11:08:05.213+0100] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-05-14T11:08:05.216+0100] {subprocess.py:93} INFO - - models.dbt_project.forPy
[2025-05-14T11:08:06.647+0100] {subprocess.py:93} INFO - [0m10:08:06  Found 8 models, 1 seed, 4 data tests, 1 source, 540 macros
[2025-05-14T11:08:06.654+0100] {subprocess.py:93} INFO - [0m10:08:06
[2025-05-14T11:08:16.594+0100] {subprocess.py:93} INFO - [0m10:08:16  Concurrency: 4 threads (target='dev')
[2025-05-14T11:08:16.600+0100] {subprocess.py:93} INFO - [0m10:08:16
[2025-05-14T11:08:16.607+0100] {subprocess.py:93} INFO - [0m10:08:16  1 of 1 START sql view model public.clean_bank_reviews .......................... [RUN]
[2025-05-14T11:08:16.922+0100] {subprocess.py:93} INFO - [0m10:08:16  1 of 1 OK created sql view model public.clean_bank_reviews ..................... [[32mCREATE VIEW[0m in 0.31s]
[2025-05-14T11:08:16.966+0100] {subprocess.py:93} INFO - [0m10:08:16
[2025-05-14T11:08:16.969+0100] {subprocess.py:93} INFO - [0m10:08:16  Finished running 1 view model in 0 hours 0 minutes and 10.31 seconds (10.31s).
[2025-05-14T11:08:17.097+0100] {subprocess.py:93} INFO - [0m10:08:17
[2025-05-14T11:08:17.102+0100] {subprocess.py:93} INFO - [0m10:08:17  [32mCompleted successfully[0m
[2025-05-14T11:08:17.104+0100] {subprocess.py:93} INFO - [0m10:08:17
[2025-05-14T11:08:17.106+0100] {subprocess.py:93} INFO - [0m10:08:17  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-05-14T11:08:20.073+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-14T11:08:20.139+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_bank_reviews, task_id=run_clean_reviews, execution_date=20250513T234436, start_date=20250514T100716, end_date=20250514T100820
[2025-05-14T11:08:20.186+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-14T11:08:20.239+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
